### PR TITLE
The setup docs have wrong types in the annotations

### PR DESCRIPTION
**Pull request by tomcoonen**
`bool` should be `self`, fixed!

### CODE REVIEW

**Comment by Gummibeer** [comment]
Hey,
you are partially right  so thanks for detecting it!
The `isXyz()` methods return `bool` - they check if the passed in value is instance of the `xyz` enum instance. But you are right that the first block `@method static self monday()` should return `self` instead of `bool`.

File: docs/usage/enum-setup.md [hunk 1]
```diff
@@ -11,21 +11,21 @@ The most basic setup to define an enum by using php-doc annotations.
...
 
 /**
- * @method static bool monday()
- * @method static bool tuesday()
- * @method static bool wednesday()
- * @method static bool thursday()
- * @method static bool friday()
- * @method static bool saturday()
- * @method static bool sunday()
+ * @method static self monday()
```
Comment (Gummibeer) [inline docs/usage/enum-setup.md]: ✔️

File: docs/usage/enum-setup.md [hunk 2]
```diff
@@ -11,21 +11,21 @@ The most basic setup to define an enum by using php-doc annotations.
...
+ * @method static self sunday()
  *
- * @method static bool isMonday(int|string $value = null)
- * @method static bool isTuesday(int|string $value = null)
- * @method static bool isWednesday(int|string $value = null)
- * @method static bool isThursday(int|string $value = null)
- * @method static bool isFriday(int|string $value = null)
- * @method static bool isSaturday(int|string $value = null)
- * @method static bool isSunday(int|string $value = null)
+ * @method static self isMonday(int|string $value = null)
```
Comment (Gummibeer) [inline docs/usage/enum-setup.md]: 🚫

File: docs/usage/enum-setup.md [hunk 3]
```diff
@@ -51,13 +51,13 @@ use Spatie\Enum\Enum;
...
  * @method static self sunday()
  *
- * @method static bool isMonday(int|string $value = null)
- * @method static bool isTuesday(int|string $value = null)
- * @method static bool isWednesday(int|string $value = null)
- * @method static bool isThursday(int|string $value = null)
- * @method static bool isFriday(int|string $value = null)
- * @method static bool isSaturday(int|string $value = null)
- * @method static bool isSunday(int|string $value = null)
+ * @method static self isMonday(int|string $value = null)
```
Comment (Gummibeer) [inline docs/usage/enum-setup.md]: 🚫

File: docs/usage/enum-setup.md [hunk 4]
```diff
@@ -92,13 +92,13 @@ So if you use, for example, `time()` for the index you can only make the enum by
...
 
 /**
- * @method static bool isMonday(int|string $value = null)
- * @method static bool isTuesday(int|string $value = null)
- * @method static bool isWednesday(int|string $value = null)
- * @method static bool isThursday(int|string $value = null)
- * @method static bool isFriday(int|string $value = null)
- * @method static bool isSaturday(int|string $value = null)
- * @method static bool isSunday(int|string $value = null)
+ * @method static self isMonday(int|string $value = null)
```
Comment (Gummibeer) [inline docs/usage/enum-setup.md]: 🚫

**Comment by tomcoonen** [comment]
@Gummibeer totally right, search & replace a bit to enthusiastic ☕️

**Comment by Gummibeer** [comment]
Thanks! :)

**Comment by Gummibeer** [comment]
It's deployed :)
https://docs.spatie.be/enum/v2/usage/enum-setup/

**Pull request merged**